cEP-0019.md

```diff
@@ -0,0 +1,476 @@
+                    "author": {
+                      "login": "li-boxuan",
+                      "url": "https://github.com/li-boxuan"
+                    },
+                    "reactions": {
+                      "totalCount": 0,
+                      "nodes": []
+                    }
+                  }
+                ]
+              }
+            },
+            {
+              "bodyText": "",
+              "comments": {
+                "nodes": [
+                  {
+                    "bodyText": "You made the your own getTime function below.",
+                    "author": {
+                      "login": "blazeu",
+                      "url": "https://github.com/blazeu"
+                    },
+                    "reactions": {
+                      "totalCount": 1,
+                      "nodes": [
+                        {
+                          "createdAt": "2018-05-07T03:09:08Z",
+                          "content": "LAUGH",
+                          "user": {
+                            "login": "li-boxuan"
+                          }
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            },
+            {
+              "bodyText": "",
+              "comments": {
+                "nodes": [
+                  {
+                    "bodyText": "Can be turned into arrow function and put at the very top.",
+                    "author": {
+                      "login": "blazeu",
+                      "url": "https://github.com/blazeu"
+                    },
+                    "reactions": {
+                      "totalCount": 1,
+                      "nodes": [
+                        {
+                          "createdAt": "2018-05-05T02:19:08Z",
+                          "content": "THUMBS_UP",
+                          "user": {
+                            "login": "li-boxuan"
+                          }
+                        }
+                      ]
+                    }
+                  }
+                ]
+              }
+            }
+          ]
+        },
+        "comments": {
+          "nodes": [
+            {
+              "bodyText": "To be honest, I don't understand why we need fetch-issues.js. From the relevant code, I can see fetch-issues.js does some pre-fetching and initial-data.js dumps it into the database. But what's the aim of doing that? It doesn't seem to save API hits. These pre-fetched issues in database are replaced later anyway.",
+              "author": {
+                "login": "li-boxuan",
+                "url": "https://github.com/li-boxuan"
+              },
+              "reactions": {
+                "totalCount": 0,
+                "nodes": []
+              }
+            }
+          ]
+        }
+      }
+    }
+  }
+}
+```
+
+The collection of meta-reviews includes two parts:
+
+##### Pre-fetch meta-reviews
```

Comment:
I think all of these sub-sections are unnecessary.
It will be better as one paragraph.

Pre-fetching could be moved to a different section about performance user requirements of the meta-review system.